sources/packages/api/src/utilities/eductaion-program-offering-utils.ts

```diff
@@ -1,6 +1,26 @@
 import { EducationProgramOffering } from "../database/entities";
 import { dateString } from ".";
 
+/**
+ * Utility to get year of study label.
+ * @param yearId
+ * @returns Year of Study label
+ */
+export const getYearOfStudyLabel = (yearId: number): string => {
+  const yearOfStudy = [
+    "Year 1",
+    "Year 2",
+    "Year 3",
+    "Year 4",
+    "Year 5",
+    "Year 6",
+    "Year 7",
+    "Year 8",
+    "Year 9",
+  ];
+  return yearOfStudy[yearId - 1];
```

Comment:
we can reduce LOC, and do something like this too ``` return `Year ${yearId}` ``` right?. just a thought

Reply:
That was good thinking. Done.